cms/templates/js/publish-xblock.underscore

```diff
@@ -101,6 +101,10 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
             </a>
             </li>
         </ul>
+        <p>
+            <%- gettext("Note: Do not hide graded assignments using this feature.") %>
```

Comment:
It should be clear contextually, but I'm open to suggestions here.

Also @catong - I have a dummy doc link in place here, it'll need to be updated to point to real documentation before this merges. Any suggestions on where it should link?

![screen shot 2016-07-19 at 11 00 57 am](https://cloud.githubusercontent.com/assets/7373924/16954800/41dd2bb2-4da0-11e6-83b4-538484252c90.png)